packages/block-library/src/navigation-menu/block-colors-selector.js

```diff
@@ -20,7 +20,7 @@ import { ColorPaletteControl, ContrastChecker } from '@wordpress/block-editor';
 const ColorSelectorIcon = ( { style } ) =>
 	<div className="block-library-colors-selector__icon-container">
 		<div
-			className="block-library-colors-selector__state-selection wp-block-navigation-menu-item"
+			className="block-library-colors-selector__state-selection"
```

Comment:
I think it might be good to make the icon an SVG. As it is, I'm not sure there's any way to guarantee which font will be used here, and an SVG might be a better way to ensure the `Aa` is correctly centred.

Reply:
can we handle this in another PR? Glad to make an issue.

Reply:
Resolve this conversation in favor of https://github.com/WordPress/gutenberg/issues/18197

Reply:
Separate PR sounds good, thanks for making the issue. 👍